### ISSUE TITLE
Allow for longer Text for Entities on homepage

### ISSUE THREAD

**Issue by fbbln** [Blackymas/NSPanel_HA_Blueprint#2020]
### Enhancement Summary

Homeage entities should allow more than 10 chars text

### Detailed Description

Currently entities can only show 10chars on the homepage.
This limits usability quite a lot.
This should be either configurable or simply not cut text and require the user to take care that nothing is overlapping.

**Comment by edwardtfn** [comment]
This is not as simple as itlooks like. The Nextion display used on those panels imposes a lot of those limitations, requiring to define fix space for each component, so we don't have all the freedom to adjust based on what an user decides to put on the screen... It must be a one solution to fit all. Also, there's a limit on Nextion memory which is defined by the number or chars on each component and it is not uncommon that we extrapolate this limit and have to find places to restrict to be able to compile the screen file.
But anyhow, we do have some space to increase number of chars on the Home page for some components. Which one is more critical for you?

**Comment by fbbln** [comment]
Entity 2 and 3 on the Homepage would be a great to show more text.
It may also be a solution to remove the icon in favor of more text as it's quite small and non-dynamic anyway.

**Comment by edwardtfn** [comment]
I've added a bit more space.
It still on `dev` only, but will be included in the next release.
Please let me know after your update if we have to work more at this.

**Comment by fbbln** [comment]
Looks better at first glance. How many chars are possible now?

**Comment by edwardtfn** [comment]
It's hard to answer that, but before there was a hard cut on 10 chars, and this was changed to 30 chars, however, there is a limit of pixels also. I have also increased that a bit, but it's hard to increase more as Nextion display cannot handle nicely when two components are overlapping in the screen.
The bumper of pixels used by a single char will be variable. An "i" needs way less than a "W", so it will depend on your used chars, but never more than 30 chars.
